libs/prelude/Prelude/Basics.idr

```diff
@@ -39,6 +41,12 @@ public export %inline
 (.) : (b -> c) -> (a -> b) -> a -> c
 (.) f g = \x => f (g x)
 
+||| Composition of a two-argument function with a single-argument one.
+||| `(.:)` is like `(.)` but the second argument and the result are two-argument functions.
+public export %inline
```

Comment:
You could perhaps add the keyword "blackbird" so that people have an easy way
to find e.g. Amar Shah's talk if they want to know more.

Reply:
The funny thing is that originally I added this as a comment but later thought that it's not a technical thing. Still, I've added a line to doc.